cmd/web.go

```diff
@@ -577,6 +581,18 @@ func runWeb(ctx *cli.Context) {
 		err = server.ListenAndServeTLS(setting.CertFile, setting.KeyFile)
 	case setting.FCGI:
 		err = fcgi.Serve(nil, m)
+	case setting.UNIX_SOCKET:
+		os.Remove(listenAddr)
+		listener, err := net.Listen("unix", listenAddr)
+		if err != nil {
+			break
+		}
+		defer listener.Close()
```

Comment:
if he calls `listener.Close()` here, the listener will never listen!? Do you mean he can omit the closing?